ui/src/main/res/layout/layout_xltunai_status.xml

```diff
@@ -0,0 +1,127 @@
+        android:paddingLeft="@dimen/eight_dp"
+        android:paddingTop="@dimen/sixteen_dp"
+        android:text="@string/text_hint_xl_order_id"
+        android:textColor="@color/dark_gray"
+        android:textSize="@dimen/secondary_text_size" />
+
+    <LinearLayout
+        android:gravity="center_vertical"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+        <com.midtrans.sdk.ui.widgets.DefaultTextView
+            android:id="@+id/text_order_id"
+            android:layout_width="0dp"
+            android:background="@color/light_gray"
+            android:layout_weight="1"
+            android:layout_marginRight="@dimen/four_dp"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center"
+            android:padding="@dimen/eight_dp"
+            android:textColor="@color/black"
+            android:textSize="@dimen/header_text_size"
+            tools:text="@string/text_company_code_here" />
+
+        <com.midtrans.sdk.ui.widgets.FancyButton
+            android:id="@+id/btn_copy_order_id"
+            android:layout_width="wrap_content"
+            android:layout_height="match_parent"
+            android:layout_gravity="center_vertical"
+            android:paddingLeft="@dimen/sixteen_dp"
+            android:paddingRight="@dimen/sixteen_dp"
+            android:visibility="visible"
+            app:fb_borderColor="?attr/colorPrimary"
+            app:fb_borderWidth="1dp"
+            app:fb_defaultColor="@android:color/white"
+            app:fb_focusColor="@color/colorAccentLight"
+            app:fb_iconColor="?attr/colorPrimary"
+            app:fb_radius="5dp"
+            app:fb_text="@string/copy"
+            app:fb_textColor="?attr/colorPrimary" />
+
+    </LinearLayout>
+
+    <com.midtrans.sdk.ui.widgets.DefaultTextView
+        android:layout_marginTop="@dimen/sixteen_dp"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/text_hint_xl_merchant_code"
+        android:layout_marginLeft="@dimen/eight_dp"
+        android:textColor="@color/dark_gray"
+        android:textSize="@dimen/secondary_text_size" />
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+        <com.midtrans.sdk.ui.widgets.DefaultTextView
+            android:id="@+id/text_merchant_code"
+            android:layout_width="0dp"
+            android:layout_weight="1"
+            android:layout_height="wrap_content"
+            android:layout_marginRight="@dimen/four_dp"
+            android:layout_gravity="center"
+            android:padding="@dimen/eight_dp"
+            android:background="@color/light_gray"
+            android:textColor="@color/black"
+            android:textSize="@dimen/header_text_size"
+            tools:text="@string/text_billpay_code_here" />
+
+        <com.midtrans.sdk.ui.widgets.FancyButton
```

Comment:
You can use style `buttonSecondary` so you can remove some properties that already defined `fb_borderWidth`, `fb_focusColor`, `fb_defaultColor`, `fb_radius`.

Also you don't need `fb_iconColor`.